lib/graphiti/stats/payload.rb

```diff
@@ -31,16 +31,19 @@ def generate
             stats[name] = {}
 
             each_calculation(name, calculation) do |calc, function|
-              args = [@scope, name]
```

Comment:
Isolated this logic in method `calculate_stat` so it can be used from pagination to calculate count